src/main/java/kitchenpos/menu/dto/MenuRequest.java

```diff
@@ -0,0 +1,44 @@
+package kitchenpos.menu.dto;
+
+import kitchenpos.menu.domain.Menu;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MenuRequest {
+    private String name;
+    private BigDecimal price;
+    private Long menuGroupId;
+    private List<MenuProductRequest> menuProducts;
+
+    public MenuRequest(String name, BigDecimal price, Long menuGroupId, List<MenuProductRequest> menuProducts) {
+        this.name = name;
+        this.price = price;
+        this.menuGroupId = menuGroupId;
+        this.menuProducts = menuProducts;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public BigDecimal getPrice() {
+        return price;
+    }
+
+    public Long getMenuGroupId() {
+        return menuGroupId;
+    }
+
+    public List<MenuProductRequest> getMenuProducts() {
+        return menuProducts;
+    }
+
+    public Menu toEntity() {
+        return new Menu(name, price, menuGroupId,
+                menuProducts.stream()
+                    .map(menuProductRequest -> menuProductRequest.toEntity())
```

Comment:
```suggestion
                    .map(MenuProductRequest::toEntity)
```
이 부분도 이렇게 표현 가능 할 거 같습니다

Reply:
만약 수정하신다면 전체적으로 다 한 번 확인해보셔도 좋을 거 같아요!